yesod-auth/Yesod/Auth/Hardcoded.hs

```diff
@@ -160,10 +160,14 @@ authHardcoded =
   where
     dispatch "POST" ["login"] = postLoginR >>= sendResponse
     dispatch _ _ = notFound
-    loginWidget toMaster =
+    loginWidget toMaster = do
+      request <- getRequest
+      let tokenKey = ("_token" :: Text) -- This value taken from yesod-form's postHelper. Not ideal that it's hard-coded in two places.
```

Comment:
I think we can use [defaultCsrfParamName](http://haddock.stackage.org/lts-5.3/yesod-core-1.4.19/Yesod-Core-Handler.html#v:defaultCsrfParamName) in both places instead.